wdl-lint/tests/lints/missing-blank-line-after-version/source.wdl

```diff
@@ -1,8 +1,12 @@
-#@ except: BlankLinesBetweenElements, DescriptionMissing
+#@ except: DescriptionMissing, Todo
+
 ## This is a test of a missing blank line following the version statement.
+## TODO: this emits two errors, one from VersionFormatting and one from
```

Comment:
File `TODO` issue? I assume the `BlankLinesBetweenElements` rule needs to ignore the version statement when emitting diagnostics.